.github/workflows/ci.yml

```diff
@@ -0,0 +1,115 @@
+name: ci-workflow
+
+# actors
+  # source repo: official terratest repo (gruntwork-io/terratest)
+  # forked repo: (e.g., xyz/terratest, abc/terratest)
+  # pr: created from forked repo, against source repo (gruntwork-io/terratest/pull/{PR NUMBER})
+# flow
+  # developer fork the source repo
+  # developer creates a branch in the forked repo
+  # developer does the development 😎
+  # developer creates a pr
+  # this pipeline will be triggered automatically, with the following input values
```

Comment:
Is it triggered automatically or manually? I thought `workflow_dispatch` was for manually triggered workflows?

Reply:
yes, `workflow_dispatch` trigger type is for manually triggered workflows. I added a couple of lines to the comment section to make the flow more clear. webhook on source repo is triggering a webapp, webapp triggers this pipeline automatically. I hope this clarifies the flow. let me know if it needs to be improved.

Reply:
Ah, OK, that's very helpful, thanks!